backend/vkt/src/main/java/fi/oph/vkt/model/Person.java

```diff
@@ -25,10 +26,21 @@ public class Person extends BaseEntity {
   private long id;
 
   @Size(max = 255)
-  @Column(name = "identity_number", nullable = false, unique = true)
+  @Column(name = "oid", unique = true)
+  private String oid;
+
+  @Size(max = 255)
+  @Column(name = "identity_number", unique = true)
   private String identityNumber;
 
-  @Column(name = "last_name", nullable = false)
+  @Column(name = "date_of_birth", unique = true)
+  private LocalDate dateOfBirth;
+
+  @Size(max = 2048)
+  @Column(name = "other_identifier")
```

Comment:
Tää muuten varmaan uniikki ja tälle pitäis lisätä kannan tasolle myös unique constraint?